Automattic/jetpack
# AI Proofread: Update Breve toggle location

Move proofread on top of the AI Assistant panel features and update Breve toggle location

## Proposed changes:

* Put Breve toggle location with label.
* Remove check of `isPluginSidebarOpened` since it's not required.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Open your editor, nothing should change.
* Activate Breve feature.
* The toggle should live together with the `AI FEEDBACK` title.
